### ISSUE TITLE
Insufficiently obvious keyboard highlight on navbar

### ISSUE THREAD

**Issue by Floppy** [manyfold3d/manyfold#4855]
<img width="272" height="47" alt="Image" src="https://github.com/user-attachments/assets/2106e039-bb28-43ed-b21f-cb84b9e94465" />

When navigating using only a keyboard there should be a clear indicator of where you are currently at with more than just color. The part of the navigation bar in the picture does change its color to a more proficient greying white but that is the only indicator. Other places do also have an outline, which would be sufficient if combined with the color change.

**Comment by Floppy** [comment]
I think this is fixed now from other changes.
